packages/common/src/models/Stems.ts

```diff
@@ -33,7 +32,7 @@ export const stemCategoryFriendlyNames = {
 }
 
 export type StemUpload = {
-  metadata: TrackMetadata
+  metadata: TrackMetadataForUpload
```

Comment:
The real work in this migration is replacing all of the instances of our computed/decorated types with more appropriate types that are correctly scoped to the use case 😁

Reply:
yes 100%